--- hostname-0.4.1/Cargo.toml	1970-01-01T00:00:01+00:00
+++ hostname-0.4.1/Cargo.toml	2025-08-26T14:23:46.963008+00:00
@@ -70,11 +70,3 @@
 [target.'cfg(any(unix, target_os = "redox"))'.dependencies.libc]
 version = "0.2"
 
-[target.'cfg(target_os = "windows")'.dependencies.windows-link]
-version = "0.1.1"
-
-[target.'cfg(target_os = "windows")'.dev-dependencies.similar-asserts]
-version = "1.6.1"
-
-[target.'cfg(target_os = "windows")'.dev-dependencies.windows-bindgen]
-version = "0.61"
