From 4b4523ab0012776499fded376326b5fbd874ff62 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?I=C3=B1aki=20=C3=9Acar?= <iucar@fedoraproject.org>
Date: Tue, 24 Sep 2024 10:50:09 +0200
Subject: [PATCH] fix resources path

---
 src/cpp/session/SessionOptions.cpp | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/src/cpp/session/SessionOptions.cpp b/src/cpp/session/SessionOptions.cpp
index 310d049..c79e01f 100644
--- a/src/cpp/session/SessionOptions.cpp
+++ b/src/cpp/session/SessionOptions.cpp
@@ -711,13 +711,7 @@ void Options::resolveQuartoPath(const FilePath& resourcePath,
 void Options::resolveNodePath(const FilePath& resourcePath,
                               std::string* pPath)
 {
-#if defined(__linux__) && !defined(RSTUDIO_PACKAGE_BUILD)
-   // node version should match RSTUDIO_INSTALLED_NODE_VERSION
-   FilePath dependenciesPath = resourcePath.completePath("../../dependencies/common/node/20.15.1-patched");
-   resolvePath(dependenciesPath, pPath);
-#else
    resolvePath(resourcePath, pPath);
-#endif
 }
 
 void Options::resolveRsclangPath(const FilePath& resourcePath,
-- 
2.46.0

